File: Codigos estudiantes por lenguaje/PY/Luciano_Chavarria/Matemáticas/PrimeFactors.py
from sys import stdout

def PrimeFactors(n):
    i = int(2)
    while i**2 <= n:
        while n % i == 0:
            stdout.write(str(i)+"\n")
            n //= i
        i+=1
    if n>1:
        stdout.write(str(n))


def main():
    PrimeFactors(20)

if __name__ == '__main__':
    main()

File: Codigos estudiantes por lenguaje/Java/Luciano_Chavarria/Matemáticas/primeFactors.java
public class Main {
    
    static void primeFactors(int n) {
        for (long p = 2; p * p <= n; p++) {
            while (n % p == 0) {                
                System.out.println(p);
                n /= p;
            }
        }
        if (n > 1) {
            System.out.println(n);
        }
    }
    
    public static void main(String[] args) {
        primeFactors(190);
    }
}
